src/services/cherry-picker.ts

```diff
@@ -172,10 +172,10 @@ export class CherryPicker {
     elapsedTime: number,
     result: number,
     timeout?: number,
-    sessionKey?: string
+    pocketSession?: Session
   ): Promise<void> {
-    await this._updateServiceQuality(blockchain, applicationID, elapsedTime, result, 900, timeout, sessionKey)
-    await this._updateServiceQuality(blockchain, serviceNode, elapsedTime, result, 7200, timeout, sessionKey)
+    await this._updateServiceQuality(blockchain, applicationID, elapsedTime, result, 900, timeout, pocketSession)
```

Comment:
It looks like here we are still using "blockchain" when really what we mean is "blockchainID" -- is that right?  Can we get these variables to match all the way through, thanks